OpenUpSA/wazimap-ng
# Fix/display ordered point in array

## Description
Updated the point dataloader to store location attributes in a list rather than dict in order to preserve their order when saved to the database. It is possible that the order is already lost before it is passed to the loaddata function but this PR does not address that.

Also added a data migration to convert data already in the database

## Related Issue
https://github.com/OpenUpSA/wazimap-ng/issues/175

## How to test it locally
1. Upload a point csv file
2. Check  the data for a location to ensure that it is stored in a list

A sample csv file 
[labour_centres.csv.gz](https://github.com/OpenUpSA/wazimap-ng/files/5629611/labour_centres.csv.gz)

## Changelog

### Added
- data migration
### Updated
- data is now stored in a list rather than a dict

### Removed


## Checklist


### Pull Request


### Commits


### Code Quality


### Testing
